common4j/src/main/com/microsoft/identity/common/java/cache/SharedPreferencesAccountCredentialCacheWithMemoryCache.java

```diff
@@ -736,4 +747,11 @@ private static boolean isCredential(@NonNull String cacheKey) {
         return isCredential;
     }
 
+    private boolean loadSha1ClearedFlag() {
```

Comment:
>loadSha1ClearedFlag [](http://example.com/codeflow?start=20&length=19)

isSha1Cleared() is more clear I think.  load is a bit ambiguous in that the load result could succeed but the value is false.